makefile

```diff
@@ -227,13 +233,17 @@ test: requirements dev-requirements
 	fi;
 
 
+vision-gpu-tests: vision-requirements dev-requirements
+	$(PYTEST) tests/vision/gpu_tests
+
+
 test-win:
 	@test -d $(WIN_ENV) || python -m venv $(WIN_ENV)
 	@$(WIN_ENV)\Scripts\activate.bat
 	$(PIP_WIN) install -q\
 			"torch==1.10.2+cpu" "torchvision==0.11.3+cpu" \
 			-f https://s3.amazonaws.com/pytorch/whl/torch_stable.html;
-	@$(PIP_WIN) install -q "langdetect>=1.0.9" "textblob>=0.17.1";
+	@$(PIP_WIN) install -q "langdetect>=1.0.9" "textblob>=0.17.1" "fasttext>=0.8.0";
```

Comment:
remove langdetect.
Also, maybe we can just add textblob to nlp-prop-requirements and then do install -r on that file here?

Reply:
removed this line completely,

>Also, maybe we can just add textblob to nlp-prop-requirements and then do install -r on that file here?

it is already in `nlp-requirements.txt`